site/src/pages/RoadmapPage/Planner.tsx

```diff
@@ -35,21 +37,23 @@ const Planner: FC = () => {
   const dispatch = useAppDispatch();
   const [cookies] = useCookies(['user']);
   const isFirstRenderer = useFirstRender();
-  const data = useAppSelector(selectYearPlans);
+  const currentPlanData = useAppSelector(selectYearPlans);
+  const allPlanData = useAppSelector(selectAllPlans);
   const transfers = useAppSelector((state) => state.roadmap.transfers);
 
   const [missingPrerequisites, setMissingPrerequisites] = useState(new Set<string>());
 
   useEffect(() => {
     // stringify current roadmap
+    // console.log(allPlanData, currentPlanData)
     const roadmapStr = JSON.stringify({
-      planner: collapsePlanner(data),
+      planners: collapseAllPlanners(allPlanData),
       transfers: transfers,
     });
 
     // stringified value of an empty roadmap
     const emptyRoadmap = JSON.stringify({
-      planner: [defaultYear()],
+      planners: [{ name: "Peter's Roadmap", content: [defaultYear()] }],
```

Comment:
I think it would be good to have a constant for "Peter's Roadmap" since I assume it is used elsewhere to initialize the planner.